web/static/js/surveys.js

```
@@ -0,0 +1,100 @@
+      </div>
+    </div>
+  `;
+
+        // Add type change handler
+        newQuestion.querySelector('.question-type-selector').addEventListener('change', handleTypeChange);
+        document.getElementById('questions').appendChild(newQuestion);
+    });
+
+    function validateScaleValues(input) {
+        const parent = input.closest('.scale-fields');
+        const min = parseInt(parent.querySelector('input[name="scale_min[]"]').value);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)